ci/jobs/macros.yaml

```diff
@@ -67,6 +67,16 @@
                 - semyers
                 - ttereshc
 
+# Ownership that defines owners for Bugzilla/Redmine integration job
+- property:
+    name: bz-redmine-ownership
+    properties:
+        - ownership:
+            owner: dadavis
```

Comment:
I don't know how 'daviddavis' got created or how to log into it. My kerberos id though is 'dadavis' so I'd be in favor of removing 'daviddavis'.